src/Validation/Admin/ValidateCreditOrderRowsWithFeeData.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Svea\Checkout\Validation\Admin;
+
+use Svea\Checkout\Validation\ValidationService;
+
+class ValidateCreditOrderRowsWithFeeData extends ValidationService
+{
+    /**
+     * @param array $data
```

Comment:
Har sett en del metoder utan kommentar och endast @param/@return, något speciell anledning?

Reply:
@alexanderwiden95 Har enbart följt så som det var tidigare, men jag börjar peta in lite PHP Docs!